docs/modules/ROOT/pages/how-tos/quick-start.adoc

```diff
@@ -0,0 +1,66 @@
+= Quick start
+
+This how-to will guide you to a working Ceph setup using the default configuration.
+
+== Prerequisites
+
+* Cluster is already managed by Project Syn
+* At least three (3) adequately sized nodes (minimum 8vCPU, 16GB RAM) are provisioned for the storage cluster
```

Comment:
```suggestion
* At least 3 adequately sized nodes (minimum 8 vCPU, 16GB RAM) are provisioned for the storage cluster
```

Reply:
According to the Microsoft Writing Style Guide, numbers zero through nine are supposed to be spelled out, cf. https://docs.microsoft.com/en-us/style-guide/numbers#numerals-vs-words

Reply:
Yes, I agree and I would also would write it out up to twelve for a general purpose text or any thesis.
But here my argument would be that this is a technical specification and then digits might be easier to grasp or remember.

But I have no hard feeling here.

Reply:
Changed to use digit `3`